import-export-cli/integration/testutils/apiProduct_testUtils.go

```diff
@@ -94,6 +95,7 @@ func exportAPIProduct(t *testing.T, name string, version string, env string) (st
 
 	t.Cleanup(func() {
 		base.RemoveAPIArchive(t, getEnvAPIProductExportPath(env), name, version)
+		base.WaitForIndexing()
```

Comment:
No need to wait here because we are only removing the export archive file and nothing related to indexing is taking place in APIM side.